test/system/irohad_test.cpp

```diff
@@ -80,16 +105,49 @@ class IrohadTest : public AcceptanceFixture {
 
   std::string params(const boost::optional<std::string> &config_path,
                      const boost::optional<std::string> &genesis_block,
-                     const boost::optional<std::string> &keypair_path) {
+                     const boost::optional<std::string> &keypair_path,
+                     const boost::optional<std::string> &additional_params) {
     std::string res;
     config_path | [&res](auto &&s) { res += " --config " + s; };
     genesis_block | [&res](auto &&s) { res += " --genesis_block " + s; };
     keypair_path | [&res](auto &&s) { res += " --keypair_name " + s; };
+    additional_params | [&res](auto &&s) { res += " " + s; };
     return res;
   }
 
   std::string setDefaultParams() {
-    return params(config_copy_, path_genesis_.string(), path_keypair_.string());
+    return params(
+        config_copy_, path_genesis_.string(), path_keypair_.string(), {});
+  }
+
+  iroha::protocol::ToriiResponse sendDefaultTx(
```

Comment:
Please add the docstring for the method.
Among the rest, it should denote that the method tries to retrieve tx status several times until `COMMITTED` will be received.